### PR TITLE
[Rule Tuning] AWS IAM Deactivation of MFA Device

### LINKED ISSUE #1111: [Rule Tuning] AWS IAM Deactivation of MFA Device
## Link to rule
https://github.com/elastic/detection-rules/blob/main/rules/aws/impact_iam_deactivate_mfa_device.toml

## Description

Add current parameter `DeleteVirtualMFADevice`

```
event.action:(DeactivateMFADevice or DeleteVirtualMFADevice) and event.dataset:aws.cloudtrail 
and event.provider:iam.amazonaws.com and event.outcome:success
```

Potentially change rule name due to the addition of  `DeleteVirtualMFADevice` 

- `AWS MFA Removal from a IAM User`

or

- `AWS IAM Removal of MFA`

or

- `AWS IAM MFA Removal`

### PR DESCRIPTION
**Pull request by austinsonger**
## Issues

Resolves #1111 

## Summary

Added `DeleteVirtualMFADevice` 

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/discovery_post_exploitation_external_ip_lookup.toml [hunk 1]
```diff
@@ -54,7 +54,10 @@ network where network.protocol == "dns" and
         "*myipaddress.com",
         "*showipaddress.com",
         "*whatismyipaddress.com",
-        "*wtfismyip.com"
+        "*wtfismyip.com",
```
Comment (bm11100) [inline rules/windows/discovery_post_exploitation_external_ip_lookup.toml:57]: This is a different rule, this file should be removed.

File: rules/aws/impact_iam_deactivate_mfa_device.toml [hunk 2]
```diff
@@ -36,7 +36,7 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and event.action:DeactivateMFADevice and event.outcome:success
+event.action:(DeactivateMFADevice or DeleteVirtualMFADevice) and event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and event.outcome:success
```
Comment (bm11100) [inline rules/aws/impact_iam_deactivate_mfa_device.toml:39]:
```suggestion
event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and event.action:(DeactivateMFADevice or DeleteVirtualMFADevice) and event.outcome:success
```

Comment (bm11100) [reply rules/aws/impact_iam_deactivate_mfa_device.toml:39]: lets keep the same order of ops as the other AWS rules.

Comment (austinsonger) [reply rules/aws/impact_iam_deactivate_mfa_device.toml:39]: Roger.

File: rules/windows/discovery_post_exploitation_external_ip_lookup.toml (hunk 1 above)
Comment (austinsonger) [reply rules/windows/discovery_post_exploitation_external_ip_lookup.toml]: Okay I revert that specific commit.

**Review by bm11100 — APPROVED** [review]

**Comment by bm11100** [comment]
@brokensound77 do we want this to target 7.13 or 7.14?

**Review by brokensound77 — APPROVED** [review]
> @brokensound77 do we want this to target 7.13 or 7.14?

7.13 - so we can merge this to main

**Pull request merged**